src/ukify/ukify.py

```diff
@@ -858,6 +893,55 @@ def generate_keys(opts):
             pub_key.write_bytes(pub_key_pem)
 
 
+def pe_read_single_section(section, opts, name, text):
+    data = section.get_data(ignore_padding=True)
+    size = section.Misc_VirtualSize
+    h = sha256(data).hexdigest()
+    data_printed = None
+
+    tmp = {
+        'size' : size,
+        'sha256' : h,
+    }
```

Comment:
OK, so you want to return a few things. Just return a few things in a tuple. Don't create a dictionary.